README.md

```diff
@@ -53,18 +53,79 @@ go run ./main.go -nheads=5
 
 Alternatively you can use the `HYDRA_NHEADS` environment var to specify the number of heads. Note the `-nheads` flag takes precedence.
 
+### Flags
+
+```console
+Usage of hydra-booster:
+  -bootstrap-conc int
+        How many concurrent bootstraps to run (default 32)
+  -bucket-size int
+        Specify the bucket size, note that for some protocols this must be a specific value i.e. for "/ipfs" it MUST be 20 (default 20)
+  -db string
+        Datastore directory (for LevelDB store) or postgresql:// connection URI (for PostgreSQL store)
+  -disable-prov-gc
+        Disable provider record garbage collection (default false).
+  -disable-providers
+        Disable storing and retrieving provider records (default false).
+  -disable-values
+        Disable storing and retrieving value records (default false).
```

Comment:
You may want to add a line here like you do for bucket size about `/ipfs` requiring these to be false. The DHT will throw an error on construction so users will find out pretty quickly if they do the wrong thing though. Your call